src/app/components/phys-desc/phys-desc.component.scss

```diff
@@ -0,0 +1,43 @@
+.phyDesc {
+    margin-bottom: 2rem;
+}
+
+.flex-container {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
```

Comment:
Maybe it's better to avoid `flex-wrap` in this case and manage it manually. When the content is shown in column, the `flex-basis` of the children (flex-item-label and flex-item-information) must be 100%. In this way you avoid style problems.